open-ce/build_env.py

```diff
@@ -99,6 +99,7 @@ def build_env(arg_strings=None):
         os.mkdir(args.repository_folder)
 
     # Create the build tree
+    from build_tree import BuildTree
```

Comment:
This import is intentionally moved after check_if_conda_build_exists because BuildTree imports conda_build and its submodules.

Reply:
Could we have this import right under the `check_if_conda_build_exists` call, with a comment saying that the order is important, that way it's clearer?